lib/ddtrace/propagation/http_propagator.rb

```diff
@@ -8,14 +8,10 @@ module HTTPPropagator
     include Ext::DistributedTracing
 
     # inject! popolates the env with span ID, trace ID and sampling priority
-    def self.inject!(span, env)
-      headers = { HTTP_HEADER_TRACE_ID => span.trace_id.to_s,
-                  HTTP_HEADER_PARENT_ID => span.span_id.to_s }
-      if span.sampling_priority
-        headers[HTTP_HEADER_SAMPLING_PRIORITY] = span.sampling_priority.to_s
-      end
-      env.merge! headers
-      env.delete(HTTP_HEADER_SAMPLING_PRIORITY) unless span.sampling_priority
```

Comment:
I know it would be good to keep this. Any ideas how we could do this? Perhaps this would be easier once we have the a globally accessible value for the `priority_sampling` flag.